src/xz/file_io.c

```diff
@@ -39,6 +39,30 @@ static bool warn_fchown;
 
 #include "tuklib_open_stdxxx.h"
 
+#ifdef _MSC_VER
+#define close	_close
+#if _MSC_VER >= 1500
+#define lseek	_lseeki64
+#else
+#define lseek	_lseek
+#endif
+#define open	_open
+#define read	_read
+#define setmode	_setmode
+#define unlink	_unlink
+#define write	_write
+#define S_ISDIR(m)	(((m) & _S_IFMT) == _S_IFDIR)
+#define S_ISREG(m)	(((m) & _S_IFMT) == _S_IFREG)
+#endif
+
+#ifndef SSIZE_MAX
```

Comment:
`SSIZE_MAX` isn't needed anymore after the change in the master branch.

Reply:
Fixed in a new commit. Removed macro `SSIZE_MAX`.